### ISSUE TITLE
Pubs labeled as preprint by default in some cite styles

### ISSUE THREAD

**Issue by gabestein** [pubpub/pubpub#2712]
### What went wrong, step-by-step?
1. Create a Pub
2. Harvard, and sometimes APA6, will call it a preprint in the citation?

### What did you expect to happen?
It should be formatted as an article, without a preprint label.

### What URL can we use to see the issue?
https://jeed.pubpub.org/pub/model-impact-hydraulic-reconnect/release/1?readingCollection=3ae3d3bc
https://www.gabestein.com/pub/zkme9ef3/release/9

**Comment by aimeediehl** [comment]
Wondering if there's been any progress on this fix. I'm still seeing "preprint" in the citations on this Pub:

https://jeed.pubpub.org/pub/model-impact-hydraulic-reconnect/release/1?readingCollection=c0be2c20#supplementary-material

Thanks for any updates!

**Comment by aimeediehl** [comment]
Hi! Just published a new Pub today and I'm still seeing "preprint" in the citation despite having specified version of record in DOI and a custom publication date in settings. Harvard style is selected in settings. Any hope for a fix?

Citation:

Bondeson, F., Faulkner, J. and Roy, E. (2023) 'Performance of a Compost Aeration and Heat Recovery System at a Commercial Composting Facility', Journal of Ecological Engineering Design [Preprint]. doi:10.21428/f69f093e.769abde7.

Pub link:

https://jeed.pubpub.org/pub/performance-cahr-commercial-composting

Thanks and happy Thanksgiving!

**Comment by gabestein** [comment]
Hey Aimee — we don't have a fix yet, but I believe I do have a workaround for you. The citation is not tied to the DOI deposit.

Instead, it looks like, for some reason we're still trying to figure out, tge Harvard style is defaulting to a preprint for any Pub that is not in an "Issue" or "Book" type collection. The solution for this would be to make one of your two collections into an issue collection. I can do this for you in the backend if you'd like so you don't need to recreate it. Just let me know which one.

**Comment by gabestein** [comment]
Yes, or we could change on of your existing ones to an issue in the backend.

**Comment by aimeediehl** [comment]
Thanks. I'll email you directly with some specific instructions. Glad to know there's a workaround.
